app/models/financial_aid/my_awards.rb

```diff
@@ -209,18 +209,33 @@ def sub_items(award_type, item)
           alertDetails: alert_details(item['item_type']),
           termDetails: disbursements(item['item_type']),
           awardMessage: item['award_message'],
+          authFailedMessage: auth_failed_message(item['item_type']),
           itemTypeDescr: item['title']
         }
       else
         {
           alertDetails: alert_details(item['item_type']),
           remainingAmount: (item['left_col_amt'] - item['right_col_amt']).to_f,
           awardMessage: item['award_message'],
+          authFailedMessage: auth_failed_message(item['item_type']),
           itemTypeDescr: item['title']
-        } 
+        }
       end
     end
 
+    def auth_failed_message_exists?(item_type)
+      exists ||= EdoOracle::FinancialAid::Queries.get_auth_failed_message(@uid, my_aid_year, item_type)
+      exists ? true : false
```

Comment:
@marcpechaitis, I understand that this query either returns a falsey value such as `nil` or `0`, or an actual response (which indicates true). It looks like you're just trying to convert this to an actual `true` or `false` value. I think you could do this by using `!!exists`

Reply:
thanks for the tip @redconfetti !